erpnext/patches/v12_0/rename_bank_reconciliation.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2018, Frappe and Contributors
```

Comment:
Add this patch in patches.txt

Reply:
Also need to delete Bank Reconciliation from the doctype
frappe.delete_doc("DocType", "Bank Reconciliation")

Doctype is still present even after running the patch on the local system 
<img width="1213" alt="Screenshot 2020-04-15 at 6 40 30 PM" src="https://user-images.githubusercontent.com/8780500/79340968-af154100-7f48-11ea-9348-c23a84155064.png">